New Features and Removing Deprecated.

pacollins opened the pull request:
- Added itemized template/archetype

- Adjust _About_ and _Contact_ page structure

- Added 404, 504, and Maintenance Pages. _Note:_ 504 and Maintenance pages won't build with Hugo (as far as I know), but now they exist for someone to use if they want (Just used a generator and matched the colors)

- Removed all minified files and references since they are outdated for theme

- Dynamic header title to match section of website you are on

- Lastly, updated the example site to show these features

Also, #7 put customCSS to customcss - I couldn't find that on the linked page, but if we are doing that, it need to change in partials/header.html, partials/footer.html (customJS), and the config.toml.

pacollins committed:
Merge pull request #2 from hjiawei/master

Update To Hjiawei Master

pacollins committed:
Inclusion of "Itemized" Page Type, Dynamic Navbar Title, and CSS Updates

pacollins committed:
New Features and Removing Deprecated.

Added itemized template/archetype, adjust about and contact page structure, added 404, 504, and maintenance pages (504/maint won't load with hugo, but they exist for someone to use), removed all minified files as they are outdated for HFI theme, dynamic header title to match section of website you are on, and lastly, updated the example site to show these features.

pacollins committed:
Generic Type rather than my personal one.

pacollins committed:
Patch 1

pacollins committed:
Add Itemized to Menu

pacollins committed:
Dynamic Titles Toggle in config.toml

hjiawei commented:
@pacollins Thanks for your PR and I really appreciate your hard work to make this template better. I created my fork for my own blogger and made some quick fixes to upgrade third-party libraries. However, I am not really familiar with the code base. Since this changeset is kinda large, I am afraid I can't do a proper review and resolve the conflicts in the PR. I am not saying PRs are not welcome but, if possible, could you please separate this into smaller sets and/or include people from other forks to the review? Again, thanks!

pacollins commented:
@Hjiawei - Sure. It makes it more difficult since the base is inactive. I just tagged onto yours since it was the most active originally. Some of these were intertwined changes, hence the massive set of commits. I am looking at adding some automated checks to my fork.

For what its worth, layouts/contact/single.html got removed to become layouts/contact/list.html, and I'm not sure why the layouts/partials/header.html is a conflict - it is pure edits so the resolution is just claiming the branch. Let me see what I can do. By including other forks, do you mean make PR there (I'm getting comfortable with Hugo and this theme, but some of the more proper git protocols goes over my head)?

pacollins committed:
Merge branch 'master' into pacollins

pacollins committed:
Update shortcode platform to format. More universal.

pacollins committed:
Update (#18)

* Adjustments for #4

* Adjustment for #4

Patch-1

* Disable RSS Feed Icon

stefanwalther/hugo-future-imperfect:rss-icon-show-option

Port since the merge was having issues.

* Update Task List

* Formatting

* Inclusion of Fancybox

Gallery images and individual images.

* Add Post View Styling

* Additional Fancybox CSS

* New Task - Hide Email

* Update Docs (#11)

* Cleaning CSS for more manageable theme development. (#10)

* Intergration with Netlify Forms. (#9)

* Inclusion of Staticman (staticman.net) for fully static comments. (#8)

* Update Theme

* Revert "Update Theme"

This reverts commit f0415e77aa1b0c17cb7adff3e7367d6c7d19bb8c.

* Update shortcode platform to format. More universal. (#12)

* Update Sidebar to Match HTML5Up Original (#13)

* Updating the sidebar to be an appropriate port.

* Fix recent post formatting

* Clean HTML Formatting, single tab instead of double. (#14)

* Reorder Social Icons for better UX. Additional Communities added. (#15)

* Patch

* Fix customCSS and customJS parameters (#49)

All site-level configuration keys are stored as lower case in Hugo

* Update shortcode platform to format. (#50)

More universal.

* Sidebar Update (#51)

* Updating the sidebar to be an appropriate port.

* Fix recent post formatting

* Update social aspects of site (#53)

Reorder Social Icons for better UX. Additional Communities added.

* Clean HTML (#54)

* Clean HTML Formatting, single tab instead of double.
